core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -33,6 +33,31 @@
 #include <sm/teesmc_opteed_macros.h>
 #include <sm/teesmc_opteed.h>
 
+	/*
+	 * Setup SP_EL0 and SPEL1, SP will be set to SP_EL0.
+	 * SP_EL0 is assigned stack_tmp + (cpu_id + 1) * stack_tmp_stride
+	 * SP_EL1 is assigned thread_core_local[cpu_id]
+	 */
+	.macro set_sp
+		bl	get_core_pos
+		cmp	x0, #CFG_TEE_CORE_NB_CORE
+		/* Unsupported CPU, park it before it breaks something */
+		bge	unhandled_cpu
+		add	x0, x0, #1
+		adr	x2, stack_tmp_stride
+		mov	x1, #0
```

Comment:
Shouldn't be needed, from TRM
*When the data size is 32 bits, the lower 32 bits of the register are used and the upper 32 bits are ignored on a read
and cleared to zero on a write.*